tox.ini

```diff
@@ -1,22 +1,20 @@
 [tox]
-envlist = py34, py35, py36, py37, flake8
+envlist = py36, py37, flake8
 
 [travis]
 python =
     3.7: py37
     3.6: py36
-    3.5: py35
-    3.4: py34
 
 [testenv]
 passenv=HOME
 sitepackages = False
-deps = pytest
-commands = pytest
+whitelist_externals = python
+commands = python setup.py test
 
 [testenv:flake8]
 passenv=HOME
 sitepackages = False
 deps = flake8
 commands =
-    flake8 --ignore=E501 setup.py toolchest tests
+    flake8 --ignore=E501,W504 setup.py toolchest tests
```

Comment:
Why do we need W504 here? Can we not meet the convention for some reason?  Also, based on irc chat, I think you meant W503

Reply:
We do want W504 here since that is one of the issues detected in the rpmutil tests. If we change to https://lintlyci.github.io/Flake8Rules/rules/W503.html then we will see W503 issues on the same lines. We could change the tests to not have either issue, but that was a larger refactor then I wanted to do at this time.